web/src/components/Meditor/Meditor.vue

```diff
@@ -1,222 +1,227 @@
 <template>
-  <v-card
-    v-if="schema && model"
-    v-page-title="model.name"
-    class="overflow-hidden"
+  <v-dialog
```

Comment:
FYI the github diff view makes this look like a big change, but the only substantial change in this template is the entire thing is now wrapped in the `v-dialog` (which was moved from `DandisetActions.vue`).

Reply:
Random thought to enhance communication about this kind of change: have one commit that just wraps the component in the `<v-dialog>`, but doesn't do the correct indentation, then the next commit with the indentation changes. Then you can link to each commit independently to demonstrate that this is actually a small change.

Reply:
I actually disagree with this, since that would purposely leave the first commit in an incorrect state, and could cause confusion. IMO this is just an unfortunate reality of these sorts of changes.

Reply:
Yeah, I agree with that assessment as well. It depends on if you want each commit to be a logical expression of some bit of change, or a complete and correct state all on its own.